pkg/version/version.go

```diff
@@ -63,6 +62,10 @@ func match(expectedVersion, actualVersion string) error {
 			actual, expected)
 	}
 
-	return fmt.Errorf("is running version %s but the latest %s version is %s",
-		actual.version, actual.channel, expected.version)
+	if actual.version != expected.version {
```

Comment:
in the case where the version number happens to be the same but the channel is different, e.g. foo-1.2.3 vs bar-1.2.3.  would this consider those to be equal?

Reply:
The check above this one, on line 60, is checking to see if the channels match. So this function would error out with "mismatched channels" before reaching this line.